src/ml_tooling/utils.py

```diff
@@ -69,7 +74,7 @@ def find_estimator_file(path: str) -> pathlib.Path:
     return newest_match
 
 
-def _get_estimator_name(clf) -> str:
+def _get_estimator_name(clf: BaseEstimator) -> str:
```

Comment:
This assumes that we would always use a Sklearn based estimator. Isn't that to much to assume?

Reply:
It's a typehint, so it doesn't change the API, but so far we're depending on Sklearn API - our custom models would also inherit from BaseEstimator. 

If we want to switch to another framework, we would have to change the entire API anyways